android/upload_sourcemap.sh

```diff
@@ -39,6 +39,25 @@ else
     echo "Instabug: Generating sourcemap files..."
     IS_HERMES=$(grep "enableHermes:" ./android/app/build.gradle -m 1)
     if [[ $IS_HERMES == *"true"* ]]; then
+        #Find HERMES OS bin directory
+        case "$OSTYPE" in
+            darwin*)  HERMES_OS_BIN='osx-bin' ;; 
+            linux*)   HERMES_OS_BIN='linux64-bin' ;;
+            msys*)    HERMES_OS_BIN='win64-bin' ;;
+            *)        echo "unknown: $OSTYPE" ;;
+        esac
+
+        #Find HERMES command file name
+        RN_VERSION=$(npm view react-native version)
+        VERSION_NO=$( echo $RN_VERSION | cut -d "." -f2 )
```

Comment:
Let's clarify it a bit more since version and version number are both synonyms to the same thing. How about INSTALLED_RN_VERSION_MAJOR?

Reply:
Makes sense, will do.